src/mito2/src/cache/write_cache.rs

```diff
@@ -77,11 +81,74 @@ impl WriteCache {
         request: SstUploadRequest,
         write_opts: &WriteOptions,
     ) -> Result<Option<SstInfo>> {
-        // TODO(yingwen): Write to the local store and then upload.
-        // Now we write to the remote and ignore local cache.
-        let mut writer =
-            ParquetWriter::new(request.upload_path, request.metadata, request.remote_store);
-        writer.write_all(request.source, write_opts).await
+        let timer = FLUSH_ELAPSED
```

Comment:
I believe we should add a fast path when using local file object storage, as it eliminates the need for the cache and upload stage.

Reply:
The write cache is disabled by default so there is already a fast path in the `AccessLayer`. I'll implement a strategy to decide whether to enable the write cache after it is ready.